### PR TITLE
chore(splitButton, tree):  Accessibility improvements 

### PR DESCRIPTION
**Pull request by kolaps33**
Purpose of the PR is polish accessibility to be ready for accessibility testing.

### CODE REVIEW

File: docs/src/examples/components/Tree/Types/TreeExample.shorthand.tsx [hunk 1]
```diff
@@ -70,6 +70,6 @@ const items = [
   },
 ]
 
-const TreeExampleShorthand = () => <Tree items={items} />
+const TreeExampleShorthand = () => <Tree aria-labelledby="default" items={items} />
```
Comment (jurokapsiar) [inline docs/src/examples/components/Tree/Types/TreeExample.shorthand.tsx]: use `aria-label` as the examples should not reference anything outside of it

File: docs/src/examples/components/Tree/Types/TreeExclusiveExample.shorthand.tsx [hunk 2]
```diff
@@ -40,6 +40,6 @@ const items = [
   },
 ]
 
-const TreeExclusiveExample = () => <Tree items={items} exclusive />
+const TreeExclusiveExample = () => <Tree aria-labelledby="exclusive" items={items} exclusive />
```
Comment (jurokapsiar) [inline docs/src/examples/components/Tree/Types/TreeExclusiveExample.shorthand.tsx]: use `aria-label` as the examples should not reference anything outside of it

File: docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx [hunk 3]
```diff
@@ -26,7 +26,23 @@ const items = [
...
+        key: items[0].key,
+        icon: items[0].icon,
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message-icon-content',
+      }}
+      onMainButtonClick={() => alert('button was clicked')}
+      toggleButton={{ 'aria-label': 'more options' }}
+    />
+    <span aria-hidden="true" id="instruction-message-icon-content" style={{ opacity: 0 }}>
+      to open menu, press Alt + Arrrow Down{' '}
```
Comment (jurokapsiar) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx]: remove `{' '}`

File: docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx [hunk 4]
```diff
@@ -26,7 +26,23 @@ const items = [
 ]
 
 const SplitButtonIconAndContentExampleShorthand = () => (
-  <SplitButton menu={items} button={items[0]} />
+  <div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx]:
```suggestion
  <>
```

File: docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx [hunk 5]
```diff
@@ -26,7 +26,23 @@ const items = [
...
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message-icon-content',
+      }}
+      onMainButtonClick={() => alert('button was clicked')}
+      toggleButton={{ 'aria-label': 'more options' }}
+    />
+    <span aria-hidden="true" id="instruction-message-icon-content" style={{ opacity: 0 }}>
+      to open menu, press Alt + Arrrow Down{' '}
+    </span>
+  </div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx]:
```suggestion
  </>
```

File: docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx (hunk 4 above)
Comment (mnajdova) [reply docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx]: If the div is not required, please change it with fragment.

File: docs/src/examples/components/SplitButton/Slots/SplitButtonToggleButtonExample.shorthand.tsx [hunk 6]
```diff
@@ -6,15 +6,28 @@ const SplitButtonExampleToggleButtonShorthand = () => {
...
+        }}
+        toggleButton={{
+          icon: open ? 'triangle-up' : 'triangle-down',
+          'aria-label': 'more options',
+        }}
+        onOpenChange={(e, { open }) => setOpen(open)}
+        onMainButtonClick={() => alert('button was clicked')}
+      />
+      <span aria-hidden="true" id="instruction-message-icon" style={{ opacity: 0 }}>
+        to open menu, press Alt + Arrrow Down{' '}
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonToggleButtonExample.shorthand.tsx]:
```suggestion
        to open menu, press Alt + Arrow Down
```

File: docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx (hunk 3 above)
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonIconAndContentExample.shorthand.tsx]:
```suggestion
      to open menu, press Alt + Arrow Down
```

File: docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx [hunk 7]
```diff
@@ -2,13 +2,24 @@ import * as React from 'react'
...
+      button={{
+        content: 'New conversation',
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message',
+      }}
+      toggleButton={{ 'aria-label': 'more options' }}
+      onMainButtonClick={() => alert('button was clicked')}
+    />
+    <span aria-hidden="true" id="instruction-message" style={{ opacity: 0 }}>
+      to open menu, press Alt + Arrrow Down
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx]:
```suggestion
      to open menu, press Alt + Arrow Down
```

File: docs/src/examples/components/SplitButton/Types/SplitButtonExamplePrimary.shorthand.tsx [hunk 8]
```diff
@@ -2,14 +2,25 @@ import * as React from 'react'
...
+        content: 'New conversation',
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message-primary-button',
+      }}
+      primary
+      toggleButton={{ 'aria-label': 'more options' }}
+      onMainButtonClick={() => alert('button was clicked')}
+    />
+    <span aria-hidden="true" id="instruction-message-primary-button" style={{ opacity: 0 }}>
+      to open menu, press Alt + Arrrow Down
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Types/SplitButtonExamplePrimary.shorthand.tsx]:
```suggestion
      to open menu, press Alt + Arrow Down
```

**Comment by mnajdova** [comment]
Please add changelog entry

File: docs/src/examples/components/SplitButton/Slots/SplitButtonToggleButtonExample.shorthand.tsx [hunk 9]
```diff
@@ -6,15 +6,28 @@ const SplitButtonExampleToggleButtonShorthand = () => {
...
-    <SplitButton
-      menu={[
-        { key: 'group', content: 'New group message' },
-        { key: 'channel', content: 'New channel message' },
-      ]}
-      button="New conversation"
-      toggleButton={{ icon: open ? 'triangle-up' : 'triangle-down' }}
-      onOpenChange={(e, { open }) => setOpen(open)}
-    />
+    <div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonToggleButtonExample.shorthand.tsx]:
```suggestion
    <>
```

File: docs/src/examples/components/SplitButton/Slots/SplitButtonToggleButtonExample.shorthand.tsx [hunk 10]
```diff
@@ -6,15 +6,28 @@ const SplitButtonExampleToggleButtonShorthand = () => {
...
+          icon: open ? 'triangle-up' : 'triangle-down',
+          'aria-label': 'more options',
+        }}
+        onOpenChange={(e, { open }) => setOpen(open)}
+        onMainButtonClick={() => alert('button was clicked')}
+      />
+      <span aria-hidden="true" id="instruction-message-icon" style={{ opacity: 0 }}>
+        to open menu, press Alt + Arrow Down
+      </span>
+    </div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Slots/SplitButtonToggleButtonExample.shorthand.tsx]:
```suggestion
    </>
```

File: docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx [hunk 11]
```diff
@@ -2,13 +2,24 @@ import * as React from 'react'
...
 
 const SplitButtonExampleShorthand = () => (
-  <SplitButton
-    menu={[
-      { key: 'group', content: 'New group message' },
-      { key: 'channel', content: 'New channel message' },
-    ]}
-    button="New conversation"
-  />
+  <div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx]:
```suggestion
  <>
```

File: docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx [hunk 12]
```diff
@@ -2,13 +2,24 @@ import * as React from 'react'
...
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message',
+      }}
+      toggleButton={{ 'aria-label': 'more options' }}
+      onMainButtonClick={() => alert('button was clicked')}
+    />
+    <span aria-hidden="true" id="instruction-message" style={{ opacity: 0 }}>
+      to open menu, press Alt + Arrow Down
+    </span>
+  </div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx]:
```suggestion
  </>
```

File: docs/src/examples/components/SplitButton/Types/SplitButtonExamplePrimary.shorthand.tsx [hunk 13]
```diff
@@ -2,14 +2,25 @@ import * as React from 'react'
...
 const SplitButtonExamplePrimaryShorthand = () => (
-  <SplitButton
-    menu={[
-      { key: 'group', content: 'New group message' },
-      { key: 'channel', content: 'New channel message' },
-    ]}
-    button="New conversation"
-    primary
-  />
+  <div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Types/SplitButtonExamplePrimary.shorthand.tsx]:
```suggestion
  <>
```

File: docs/src/examples/components/SplitButton/Types/SplitButtonExamplePrimary.shorthand.tsx [hunk 14]
```diff
@@ -2,14 +2,25 @@ import * as React from 'react'
...
+        'aria-describedby': 'instruction-message-primary-button',
+      }}
+      primary
+      toggleButton={{ 'aria-label': 'more options' }}
+      onMainButtonClick={() => alert('button was clicked')}
+    />
+    <span aria-hidden="true" id="instruction-message-primary-button" style={{ opacity: 0 }}>
+      to open menu, press Alt + Arrow Down
+    </span>
+  </div>
```
Comment (mnajdova) [inline docs/src/examples/components/SplitButton/Types/SplitButtonExamplePrimary.shorthand.tsx]:
```suggestion
  </>
```

**Review by mnajdova — APPROVED** [review]
Please see final comments before merging, and add a changelog entry.

**Pull request merged**